### ISSUE TITLE
Additional support for VTLs and styles hosted on ArcGIS Enterprise

### ISSUE THREAD

**Issue by jwasilgeo** [Esri/esri-leaflet-vector#88]
Add a `portalUrl` optional parameter to the [`L.esri.Vector.vectorTileLayer`](https://github.com/Esri/esri-leaflet-vector#lesrivectorvectortilelayer) layer type, intended to be used with the "ITEM_ID" constructor flavor.

```js
L.esri.Vector.vectorTileLayer("ITEM_ID", {
  
  // ADD NEW OPTIONAL PROPERTY
  portalUrl: "https://PATH/TO/ARCGIS/ENTERPRISE/",
  // IF NOT PROVIDED, WILL DEFAULT TO SOME PART OF "https://www.arcgis.com/sharing/rest/content/items/" 
  // SHOULD BE SIMILAR IN NATURE TO https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#portalUrl

  // provide either apiKey or token if not public
  apiKey: "...",
  token: "...",

  // optionally customize the style with a function that gets the default style from the service
  // and returns the new style to be used
  style: (style) => {
    return newStyle;
  }
}).addTo(map);
```

Review internal `Util.js` usage here:

- https://github.com/Esri/esri-leaflet-vector/blob/master/src/Util.js#L40
- https://github.com/Esri/esri-leaflet-vector/blob/master/src/Util.js#L46

NOTE: The other layer type, [`L.esri.Vector.vectorBasemapLayer`](https://github.com/Esri/esri-leaflet-vector#lesrivectorvectorbasemaplayer), will **not support** this enhancement as it 100% uses the the Esri Basemap Styles API internally.

Related issues:

- #28 
- ~#68~ (Unknown, needs more info)
- #80

**Comment by jwasilgeo** [comment]
in progress in branch [88-enterprise-support](https://github.com/Esri/esri-leaflet-vector/tree/88-enterprise-support)

**Comment by jwasilgeo** [comment]
This is available now in the recent [v3.1.0 release](https://github.com/Esri/esri-leaflet-vector/releases/tag/v3.1.0). 🎉
